app/components/admin/basic_link_post_component.rb

```diff
@@ -0,0 +1,18 @@
+module Admin
+  class BasicLinkPostComponent < ApplicationComponent
+    private attr_reader :title, :path, :name, :val
+
+    def initialize(title, path, name:, val:)
```

Comment:
Let's rename `val` to `value` in the initializer and for the attribute. I don't see the need to remove a few characters.

Reply:
yeah, I was about to use `value` but I forgot to.